libs/core/src/lib/multi-input/multi-input.component.spec.ts

```diff
@@ -241,4 +241,21 @@ describe('MultiInputComponent', () => {
         expect(component.selectedChange.emit).toHaveBeenCalled();
         expect(component.selected).toEqual([component.dropdownValues[0]]);
     });
+
+    it('should handle showAll button', async () => {
```

Comment:
What this test do is checking if calling `showAllClicked` method clears `searchTerm`.
Well this test is actually needed, because I cannot find any tests for `Show All Button`, but what new functionality actually does is Showing/Hiding `Show All` button depending on filter state.